### DISCUSSION TITLE
nvim icon display error with kitty

### DISCUSSION THREAD

**Discussion by longintlong** [ayamir/nvimdots#1331] [Q&A] [answered]
Hi all nvim experts,
i'm using kitty with nvim but some icon cannot display like this:
<img width="352" alt="image" src="https://github.com/user-attachments/assets/4a68b85c-296d-407c-829c-fe5d63fa0386">

I'm using JetBrainsMono-2.304 and follow :
1. Install the official JetBrains Mono fonts.
2. Install NerdFontsSymbolsOnly - [Nerd Fonts releases page](https://github.com/ryanoasis/nerd-fonts/releases) 
3. Add the symbol_map ... line from the FAQ like
```
font_family      JetBrainsMono
bold_font        auto
italic_font      auto
bold_italic_font auto

symbol_map U+23FB-U+23FE,U+2665,U+26A1,U+2B58,U+E000-U+E00A,U+E0A0-U+E0A3,U+E0B0-U+E0C8,U+E0CA,U+E0CC-U+E0D2,U+E0D4,U+E200-U+E2A9,U+E300-U+E3E3,U+E5FA-U+E62F,U+E700-U+E7C5,U+F000-U+F2E0,U+F300-U+F31C,U+F400-U+F4A9,U+F500-U+F8FF Symbols Nerd Font
```
My-vim version is :
NVIM v0.9.4
Build type: RelWithDebInfo

**Comment by longintlong** [comment, marked as answer]
emmmm. i'm sorry, restart my tmux session and it's  ok a lot!
thank u!
<img width="1920" alt="image" src="https://github.com/user-attachments/assets/3d2b83b1-79ac-421a-bb56-2719dc741518">

**Reply by ayamir** [reply]
U'd better use latest nvim (>= 0.10) and nerd font (v3.2.1).
